### PR TITLE
Janitorial toolbox

### PR DESCRIPTION
**Pull request by Unisol**
## About the PR
Added janitorial toolbox. It's purple and has following guaranteed items:
* screwdriver
* wrench
* plunger
* network configurator
* welder
* welding mask

Following items have low chance to spawn:
* Gin
* Rubber ducky

## Why / Balance
Burdman asked for one on discord, I asked - why not?
[Link to discussion](https://discord.com/channels/1139716325627932682/1303402658828128340)

## Technical details
Changes to non-harmony files: edited ClosetJanitorFilled entry in Resources/Prototypes/Catalog/Fills/Lockers/service.yml to add janitorial toolbox spawn to janitor's closet.

## Media
SS14 Loader_08M2o6f8UI
SS14 Loader_9ODQgKhLIN
SS14 Loader_tlsTLvYuUw

## Requirements

- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

:cl:
- add: Added purple toolbox with a little extra something to help janitor through his day

### CODE REVIEW

File: Resources/Prototypes/Catalog/Fills/Lockers/service.yml [hunk 1]
```diff
@@ -77,6 +77,7 @@
         amount: 2
       - id: Plunger
         amount: 2
+      - id: ToolboxJanitorialFilled
```
Comment (FluffMe) [inline Resources/Prototypes/Catalog/Fills/Lockers/service.yml:80]: Upstream file needs a `Harmony change` comment.

File: Resources/Textures/_Harmony/Objects/Tools/Toolboxes/toolbox_purple.rsi/meta.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    },
+    {
+      "name": "inhand-right",
+      "directions": 4
+    },
+    {
+      "name": "icon"
+    },
+    {
+        "name": "icon-open"
```
Comment (FluffMe) [inline Resources/Textures/_Harmony/Objects/Tools/Toolboxes/toolbox_purple.rsi/meta.json:22]: Over-indented. Not very important.

**Review by FluffMe — CHANGES_REQUESTED** [review]
Just couple of things:
Missing comment for licensing change to AGPL (mandatory change) and minor indentation issue in JSON (optional change).

This is an awesome addition!

**Comment by Unisol** [comment]
Tell me more about the "licensing change to AGPL", I'm not quite sure what you mean by this.

**Review by FluffMe — DISMISSED** [review]

**Review by FluffMe — APPROVED** [review]

**Review by DieselMohawk — APPROVED** [review]
Looks good. Good job.

**Pull request merged**